packages/strapi-plugin-content-manager/package.json

```diff
@@ -13,21 +13,22 @@
     "classnames": "^2.2.6",
     "codemirror": "^5.46.0",
     "draft-js": "^0.11.5",
+    "highlight.js": "^10.1.1",
     "immutable": "^3.8.2",
     "lodash": "^4.17.11",
     "pluralize": "^7.0.0",
     "react": "^16.9.0",
     "react-dom": "^16.9.0",
-    "react-intl": "4.5.0",
     "react-fast-compare": "^3.2.0",
+    "react-intl": "4.5.0",
+    "react-markdown": "^4.3.1",
```

Comment:
Should we remove `^`? Same for all packages.

@alexandrebodin what do you think?